src/css/_typography.scss

```diff
@@ -107,15 +127,29 @@ ol[class] {
   & > img {
     margin-bottom: 24px;
     margin-top: 0;
+    filter: drop-shadow(0px 2px 8px rgba(27, 25, 27, 0.08));
+    border-radius: 8px;
+  }
+
+  ul, ol {
+    margin-bottom: 24px;
   }
 
-  & > ul {
+  ul {
     list-style: inside;
   }
 
-  & > ol {
+  ol {
+    padding-left: 16px;
     list-style: decimal;
   }
+
+  a {
+    svg {
+      margin-left: 4px;
+      margin-bottom: 3px;
```

Comment:
I didn`t use flex here that align `svg` because we can have text in the link in two lines and with flex property it looked ugly